1.0/docs/devguide/properties.md

```diff
@@ -324,9 +324,10 @@ Example:
 
     });
 
-**Compatibility note:** The argument order for change handlers is currently the
-**opposite** of the order used in 0.5.
-{: .alert .alert-info }
+**Warning:**A single property observer shouldn't rely on any properties, 
+sub-properties, or paths because [the observer can be called while these 
+dependencies are undefined](#dependencies).
```

Comment:
Please replace the blind link with an attributed one. Like "the observer can be called while these 
 dependencies are undefined. See [Always include dependencies as observer arguments](#dependencies)
for details."

General idea: that an inline, context-free link on "foo" should only go to the definitive definition of "foo"--like an API reference page or concepts section.